haskell/baseDAO-ligo-meta.cabal

```diff
@@ -32,6 +32,7 @@ library
   exposed-modules:
```

Comment:
It looks like this file as well as `package.yaml` are missing some updates in the `extra-source-files`.